prometheus-install/README.md

```diff
@@ -0,0 +1,46 @@
+# Integrate Prometheus into Flink cluster in non-openshift cluster
```

Comment:
```suggestion
# Integrate Prometheus into Flink cluster
```
I don't think we need the "in non-openshift cluster" part here since these are the instructions most people would follow